.ibm/pipelines/resources/config_map/app-config-rhdh.yaml

```
@@ -17,6 +17,148 @@ app:
 dynamicPlugins:
   rootDirectory: dynamic-plugins-root
   frontend:
+    'janus-idp.backstage-plugin-dynamic-home-page':
```

Just one things: After PR #1944 is merged, this should be:

```suggestion
    red-hat-developer-hub.backstage-plugin-dynamic-home-page:
```

Everything else lgtm!